demo/mvp.sh

```diff
@@ -72,6 +72,6 @@ PGPASSWORD=np_as_spiel_pass ./artifacts/noisepage/psql -h localhost -d np_as_spi
 doit action_generation --args="--min-num-cols 1 --max-num-cols 4 --filter-tables"
 
 # Start picking indexes.
-doit action_recommendation --database_game_args="--use_microservice"
+doit action_recommendation --database_game_args="--use_microservice --record_predictions"
```

Comment:
```suggestion
doit action_recommendation --database_game_args="--use_microservice"
```

We probably do not want to record predictions by default, since as you mention this takes 1h on dev4.
(To be clear, what takes 1h? One iteration or all iterations of recommending indexes?)

Reply:
To document for the future -- this was 1 iteration.